addon/styles/_frost-link.scss

```diff
@@ -121,6 +121,7 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
   &.tertiary {
     background-color: inherit;
     color: $frost-color-blue-1;
+    margin-bottom: -7px;
```

Comment:
Agreed - I only noticed it today :(  I will add an issue for this.  I don't want to hold up this PR.